data/json/snippets/schizophrenia.json

```diff
@@ -4,41 +4,60 @@
     "category": "schizo_self_talk",
     "text": [
       "Hey, can you hear me?",
-      "Don't touch me.",
+      "I think I know what I'm doing.",
       "What's your name?",
       "I thought you were my friend.",
       "How are you today?",
       "Shut up!  Don't lie to me.",
+      "How do I bring you back?",
+      "They're all gone…",
+      "Why did you leave me?",
+      "Oh, there you are.",
+      "…and just where have you been?",
+      "You hungry?  I've got plenty of food.",
+      "You thirsty?  I've got water.",
       "Why would you do that?",
-      "Please, don't go.",
+      "Please don't go.",
       "Don't leave me alone!",
       "Yeah, sure.",
-      "No way, man.",
+      "No way man.",
```

Comment:
I'm pretty sure "No way, man" is actually the correct punctuation here.

Reply:
Grammatically, and by the loose rules of English, they're both actually correct. However inn a conversation or self speech, a comma there would imply a pause. Which to me didn't sound quite right given the context.